openbb_platform/extensions/crypto/openbb_crypto/price/price_router.py

```diff
@@ -56,3 +61,15 @@ async def historical(
 ) -> OBBject:
     """Get historical price data for cryptocurrency pair(s) within a provider."""
     return await OBBject.from_query(Query(**locals()))
+
+
+@router.command(methods=["GET"])
+async def live(
+    symbol: str = "ethbtc", lifetime: int = 10
+) -> BinanceCryptoHistoricalData:
```

Comment:
How can we make this a Provider Interface method? There could be any number of providers that have a WS connection, like here - https://site.financialmodelingprep.com/developer/docs#crypto-websocket